src/Modules/ContentBlocks/templates/Backend/Forms/ContentBlock.html.twig

```diff
@@ -0,0 +1,18 @@
+{% block translation_widget %}
```

Comment:
this shouldn't be here since it is not needed

Reply:
this file has been removed